teams/dynamic/coc-committee.yml

```diff
@@ -0,0 +1,25 @@
+# yaml-language-server: $schema=../sub-teams-schema.json
+name: null
+description: The Code of Conduct Committee
+charter: dynamic
+requirements: null
+scopes:
+    codeowners: null
+    other:
+        GitHub Account: https://github.com/conda-conduct
+        Matrix (private): https://matrix.to/#/!jLRCFFbpabFtdYvfeO:matrix.org
```

Comment:
Hm does this link also act as an invitation to the private channel?

Reply:
Yes, that appears to be the case, though it's tough to tell 100% since I'm already part of that channel. I opened this link in an incognito Chrome tab and it allowed me to open the channel in Element (there were also other app options offered) but as I said I'm already a member so I'm not sure if there would be some sort of block from me joining the channel otherwise.

Reply:
Attached are screenshots to display what happens when attempting to access a private channel URL:

![Screenshot 2024-06-17 at 10 27 57 AM](https://github.com/conda/governance/assets/28930622/f2444e24-91de-45d1-937b-a251d57fa792)

![element](https://github.com/conda/governance/assets/28930622/83276d15-f738-4754-958f-776f422f3a6a)

![Screenshot 2024-06-17 at 10 18 02 AM](https://github.com/conda/governance/assets/28930622/2639efb5-446e-4530-916a-ae0aedee9bdf)

Looks like there is a guardrail in place that prevents folks from joining private channels that they weren't invited to.